### PR TITLE
Attempted fix for known `riskmetric` weighting bug #293

### PR DESCRIPTION
**Pull request by AARON-CLARK**
@Jeff-Thompson12, hey I was thinking about adding the top 500 pkgs to the demo version of the app, and thought it'd be a good idea to put in a little workaround to fix for https://github.com/pharmaR/riskmetric/issues/293. What are your thoughts on these changes? Should I clean them up and push them to the `dev` branch as well? The alternative is that we could wait for `{riskmetric}` to solve the bug.

Also, you'll notice I allow tracking of the `database.sql` file on this branch.

### CODE REVIEW

**Review by Jeff-Thompson12 — COMMENTED** [review]
I'm leaning towards not adding the work around to `dev` for a few reasons.

1. It will just be on the development branch. Presumably this would be fixed before we push to master again. So we would be adding in code that we would end up taking out again and benefiting no one.
2. While this workaround works nicely for one package, it will require reconfiguring if we vectorize the package upload (we can't assume the same assessments will be NA for each package).
3. It's not standard. I think we need to keep `riskassessment` to using the standard `riskmetric` computations even if there is a known issue.

File: R/mod_assessmentInfo.R [hunk 1]
```diff
@@ -65,7 +65,7 @@ assessmentInfoServer <- function(id, metric_weights) {
     # Render table for Maintenance Metrics.
     output$riskcalc_weights_table <- DT::renderDataTable({
       d <- metric_weights() %>%
-        dplyr::mutate(weight = ifelse(name == "covr_coverage", 0, weight)) %>%
+        # dplyr::mutate(weight = ifelse(name == "covr_coverage", 0, weight)) %>%
```
[the remark this answers is not in the document]
Comment (AARON-CLARK) [reply R/mod_assessmentInfo.R:68]: Will shoot a tiny PR out

File: inst/sql_queries/initialize_metric_table.sql [hunk 2]
```diff
@@ -11,5 +11,5 @@ VALUES
 ('export_help',         'Documentation',     '% of documented objects',          1, 0, 'maintenance', 1),
 ('bugs_status',         'Bugs Closure Rate', '% of the last 30 bugs closed',     1, 0, 'maintenance', 1),
 ('license',             'License',           "Package's license",                0, 0, 'maintenance', 1),
-('covr_coverage',       'Test Coverage',     '% of objects tested',     1, 1, 'maintenance', 0),
+('covr_coverage',       'Test Coverage',     '% of objects tested',     1, 1, 'maintenance', 1),
```
[the remark this answers is not in the document]
Comment (AARON-CLARK) [reply inst/sql_queries/initialize_metric_table.sql:14]: Will shoot a tiny PR out to `dev`

File: database.sql
[the remark this answers is not in the document]
Comment (AARON-CLARK) [reply database.sql:1]: Yeah, what the heck is that?

**Comment by Jeff-Thompson12** [comment]
@AARON-CLARK I think you were trying to track `database.sqlite` but accidently added an empty file called `database.sql`.

**Comment by AARON-CLARK** [comment]
What an amateur.

**Pull request merged**